toolkit/governed-map/pallet/src/lib.rs

```diff
@@ -1 +1,232 @@
+//! Pallet exposing the key-value pairs sourced from the Cardano ledger as part of the Governed Map feature.
+//!
+//! # Purpose of this pallet
+//!
+//! This pallet stores the most recent state of the key-value pairs in the Governed Map on Cardano for use
+//! by other runtime components. It also exposes hooks for other components to be notified when particular
+//! key-value pair is inserted, updated or deleted.
+#![cfg_attr(not(feature = "std"), no_std)]
 
+use frame_support::pallet_prelude::*;
+use frame_system::pallet_prelude::*;
+use frame_system::WeightInfo;
+use sidechain_domain::byte_string::*;
+use sidechain_domain::*;
```

Comment:
Isn't there any penalty for wildcard import? Only use MainchainAddress is required.